docs/core/examples/longex2.py

```diff
@@ -75,12 +75,16 @@ class Numbers(basic.LineReceiver):
     """
     def lineReceived(self, line):
         try:
-            numbers = map(long, line.split())
+            numbers = [int(num.decode("ascii")) for num in line.split()]
```

Comment:
`int` can parse numbers out of byte strings, so there's no need to `.decode` here.